packages/tools/tsconfig.json

```diff
@@ -2,14 +2,14 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "baseUrl": "./",
-        "esModuleInterop": true,
-        "module": "CommonJS",
-        "moduleResolution": "Node",
-        "outDir": "./dist",
-        "paths": {
-            "@dd/tools/*": ["./src/*"],
-            "@dd/core/*": ["../core/src/*"],
-            "@dd/factory/*": ["../factory/src/*"]
+        "rootDir": "./",
+        "outDir": "./dist"
+    },
+    "ts-node": {
+        "transpileOnly": true,
+        "compilerOptions": {
+            "module": "commonjs",
+            "moduleResolution": "Node"
```

Comment:
To keep this package in check with the new `*.test.ts` files.

ts-node is still a PITA to run this, so it needs its very own little TS configuration in order to work.
Otherwise it throws a very cool `SyntaxError: Cannot use import statement outside a module` on the very first `import` statement from the very first TS file it runs.